evergreen/config.yml

```diff
@@ -876,6 +911,25 @@ tasks:
       verbose_test_output: true
   - func: "check branch state"
 
+- name: baas-network-tests
+  tags: [ "for_nightly_tests" ]
+  exec_timeout_secs: 14400
+  commands:
+  - func: "launch remote baas"
+    vars:
+      baas_branch: 1a0d1c021460d2215b21bc4f7c7f668714ce81c7
```

Comment:
Is this the hash of a commit? Will we have to keep updating it?

Reply:
can we just make this a global variable and share it with this? https://github.com/realm/realm-core/blob/30b7a29a093970a6d1a5d2cc7c86f8b8828c7dc9/evergreen/config.yml#L867

Reply:
This value was created to be the baas branch/commit "global value" in evergreen and would need to continue to be updated since we have been pinning our tests against a specific version of baas.

Are we stable/confident enough to just make this `master` and this value would still allow us to pin a version if we need it locally for testing?

Reply:
I would like to use latest master, but are we confident enough it will be green? 🙂 We could also start by running an experiment for a week.

Reply:
Updated baas to use master - I'll let it run for a bit on my branch before pushing up to master, but so far, the tests are green.